### ISSUE TITLE
TRUEHD in m2ts

### ISSUE THREAD

**Issue by almirus** [MediaArea/MediaInfo#505]
hi!
**Compression mode** in m2ts (with TrueHD)  is not detected correctly
![image](https://user-images.githubusercontent.com/16898685/100145579-963f1780-2ea9-11eb-950a-c6ca8c9008eb.png)

**Comment by JeromeMartinez** [comment]
> Compression mode in m2ts (with TrueHD) is not detected correctly

Oops, fixed, for next version.

**Comment by almirus** [comment]
@JeromeMartinez please also pay attention to the Stream size, it it also incorrect, must be larger

**Comment by JeromeMartinez** [comment]
True, I reopen the ticket for the bitrate part, will be longer to implement (not sure about how I'll show that, I know the core bitrate but I don't know the global bitrate due to TS which does not store such info, I need to reorganize a lot of code for a good and generic handling of such kind of stream)

**Comment by almirus** [comment]
@JeromeMartinez TrueHD havent the core... i could be wrong, but the container holds two different streams: AC3 (640kbps) and TrueHD (8Mbps)

**Comment by ValZapod** [comment]
BTW. 31.250 fps is wrong. That must be 1200 fps. I suppose it reads PTS from MPEG-TS, but it should real PTS from raw .thd.

**Comment by almirus** [comment]
@JeromeMartinez 
now is **Compression mode**   corret
but **Stream size** must be sum TrueHD stream+AC3(substream)Stream extension
